packages/launch/src/focused-launch/domain-details/index.tsx

```diff
@@ -62,16 +63,28 @@ const DomainDetails: React.FunctionComponent = () => {
 		} );
 	};
 
+	const fallbackSubtitleText = __(
+		'Free for the first year with any paid plan.',
+		__i18n_text_domain__
+	);
+	const newSubtitleText = __(
+		'Free for the first year with any annual plan.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Free for the first year with any paid plan' )` *ES Score: 6*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/monthly-plans-domain-step-subtitle-copy-changes)